gpcontrib/arenadata_toolkit/init_file

```diff
@@ -0,0 +1,24 @@
+-- start_matchsubs
+
+m/pg_toast_\d+/
+s/pg_toast_\d+/pg_toast_10000/
+
+m/pg_toast_\d+_index/
+s/pg_toast_\d+_index/pg_toast_10000_index/
+
+m/pg_aoseg_\d+/
+s/pg_aoseg_\d+/pg_aoseg_10000/
+
+m/pg_aovisimap_\d+/
+s/pg_aovisimap_\d+/pg_aovisimap_10000/
+
+m/pg_aovisimap_\d+_index/
+s/pg_aovisimap_\d+_index/pg_aovisimap_10000_index/
+
+m/        .*table_name        .*/
```

Comment:
And what is the purpose of this match-subs block?

IMO such special case match-subs should be inlined at test.